internal/controller/runtime/fsm/runtime_fsm_apply_clusterrolebindings.go

```diff
@@ -23,13 +23,19 @@ var (
 func sFnApplyClusterRoleBindings(ctx context.Context, m *fsm, s *systemState) (stateFn, *ctrl.Result, error) {
 	runtimeClient, err := m.RuntimeClientGetter.Get(ctx, s.instance)
 	if err != nil {
-		updateCRBApplyFailed(&s.instance)
+		// TODO: This probably should be replaced with requeue logic, as we do in other places
```

Comment:
100% agree! It should be common requeue behaviour across the whole KIM for situation when we cannot get SKR client